core/client/fs/src/main/java/alluxio/client/file/PositionReadFileInStream.java

```diff
@@ -41,13 +160,30 @@ public PositionReadFileInStream(PositionReader reader,
       long length) {
     mPositionReader = reader;
     mLength = length;
+    mCache = new PrefetchCache(
+        Configuration.getInt(PropertyKey.USER_POSITION_READER_STREAMING_MULTIPLIER), mLength);
```

Comment:
todo: the configuration value should come from `FileSystemContext.getClusterConf` instead of the global configuration.